src/ngAnimate/animateRunner.js

```diff
@@ -1,18 +1,33 @@
 'use strict';
 
-var $$rAFMutexFactory = ['$$rAF', function($$rAF) {
+var $$AnimateMutexFactory = ['$$rAF', function($$rAF) {
```

Comment:
Why do you call this a mutex? it's not quite the same as regular mutex. How about $$AnimateAsyncRun or $$AnimateAsyncExecutor?